### PR TITLE
ci: Make badges consistent, run markdownlint all .md files

### PR DESCRIPTION
**Pull request by spetrosi**
- Consistently generate badges for GH workflows in roles' RHELPLAN-146921
- Run markdownlint on all .md files

Signed-off-by: Sergei Petrosian <spetrosi@redhat.com>

### CODE REVIEW

File: .github/workflows/markdownlint.yml [hunk 1]
```diff
@@ -30,5 +30,5 @@ jobs:
       - name: Lint README.md
         uses: docker://avtodev/markdown-lint:master
         with:
-          args: README.md
+          args: --ignore=CHANGELOG.md **/*.md
```
Comment (liangwen12year) [inline .github/workflows/markdownlint.yml:33]: Maybe add a comment explaining why not linting CHANGELOG.md ?

File: .github/workflows/markdownlint.yml [hunk 2]
```diff
@@ -30,5 +30,5 @@ jobs:
       - name: Lint README.md
```
Comment (liangwen12year) [inline .github/workflows/markdownlint.yml:30]: How about changing the task name to "Lint all markdown files" ?

File: .github/workflows/woke.yml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 # yamllint disable rule:line-length
-name: Check for non-inclusive language
+name: Woke
```
Comment (liangwen12year) [inline .github/workflows/woke.yml:2]: The previous name is more descriptive as a task name IMHO.

File: README.md [hunk 4]
```diff
@@ -1,4 +1,6 @@
 # linux-system-roles/network
+![CodeQL](https://github.com/linux-system-roles/network/workflows/CodeQL/badge.svg) ![Python Unit Tests](https://github.com/linux-system-roles/network/workflows/Python%20Unit%20Tests/badge.svg)![ShellCheck](https://github.com/linux-system-roles/network/workflows/ShellCheck/badge.svg)![Ansible Lint](https://github.com/linux-system-roles/network/workflows/Ansible%20Lint/badge.svg) ![Ansible Test](https://github.com/linux-system-roles/network/workflows/Ansible%20Test/badge.svg) ![Markdown Lint](https://github.com/linux-system-roles/network/workflows/Markdown%20Lint/badge.svg) ![Woke](https://github.com/linux-system-roles/network/workflows/woke/badge.svg) 
+---
```
Comment (liangwen12year) [inline README.md:3]: The svg link for Woke is invalid. 
image

**Comment by liangwen12year** [comment]
And the CI task `Markdown Lint`  detects some errors on `README.md`.

File: .github/workflows/woke.yml (hunk 3 above)
Comment (spetrosi) [reply .github/workflows/woke.yml]: The badge is generated with tha name, so I wanted to make the badge toe have `woke` in it. Although, woke is not a widely-know tool, so maybe better have something more descriptive like `Detect non-inclusive language`

Comment (liangwen12year) [reply .github/workflows/woke.yml]: Is this possible to let the badge have a more descriptive name so that it can be consistent with github workflow ?

Comment (spetrosi) [reply .github/workflows/woke.yml:2]: Right now, the workflow name in PR CI jobs is `Woke / Detect non-inclusive language`, and the badge name is `Woke`. I think that's good enough.

Comment (liangwen12year) [reply .github/workflows/woke.yml:2]: Thanks for addressing this, I agree with the current change.

**Pull request merged**